ci-operator/config/openshift/ansible-operator-plugins/openshift-ansible-operator-plugins-main.yaml

```diff
@@ -75,6 +89,12 @@ tests:
       CHECK_MOD_LIST: "false"
     test:
     - ref: go-verify-deps
+- as: verify-requirements
+  commands: |
+    git diff --exit-code
+  container:
+    from: get-requirements
+  run_if_changed: images/ansible-operator/Pipfile*
```

Comment:
We should also be watchful for changes done in `openshift/install-ansible.sh` and `openshift/requirements*.txt` files since these are the actual ones that go inside the base image. With this, we'll catch any manual modifications done in these files.

Reply:
The Dockerfile.requirements also needs to be watched as it will impact the generation of the requirements files